rest-api-spec/src/main/resources/rest-api-spec/test/indices.exists_template/10_basic.yml

```diff
@@ -23,6 +25,8 @@ setup:
             number_of_replicas: 0
 
   - do:
+      allowed_warnings:
+        - "Deprecated parameter [master_timeout] used. To promote inclusive language, please use [cluster_manager_timeout] instead. It will be unsupported in a future major version."
```

Comment:
in the next major version 3.0

Reply:
@anasalkouz Thanks for this idea. My concern is once the deprecated parameter can not be removed in version 3.0, the warning message will become inaccurate.
I'm just worrying about there will be resistance of the old usage removal from the users, which make the removal postponed.
If you think it's better to specify the version number in the warning message, I will change it to `It will be removed in 3.0.`
The message is defined in this file https://github.com/opensearch-project/OpenSearch/blob/1.3.1/server/src/main/java/org/opensearch/rest/action/cat/RestNodesAction.java#L87, I will create a separate PR to change it.

Reply:
Changed the deprecation in a separate PR https://github.com/opensearch-project/OpenSearch/pull/2863

Reply:
Changed the deprecation message in YAML rest api test in commit https://github.com/opensearch-project/OpenSearch/pull/2678/commits/07fc06d5a69106078d5467993752693a64423fb0 👍